packages/scalecube-microservice/src/index.ts

```diff
@@ -1,2 +1,5 @@
-export { Microservices } from './Microservices/Microservices';
-export { ASYNC_MODEL_TYPES } from './helpers/constants';
+import { Microservices } from './Microservices/Microservices';
+import { ASYNC_MODEL_TYPES } from './helpers/constants';
+
+export { Microservices, ASYNC_MODEL_TYPES };
+export default Microservices;
```

Comment:
don't like this approach, have described above why. Is there any special reason to to it like this?